rfc/0067-extended-sdk-observability-rto-v2.md

```diff
@@ -592,7 +596,7 @@ Should be placed on all operations which are at the collection level. The attrib
 
 #### Attribute: Statement
 
-This attribute is a standard OpenTelemetry attribute and should be placed on N1QL and analytics operations. The attribute key is "`db.statement`".
+This attribute is a standard OpenTelemetry attribute and should be placed on N1QL and analytics operations. The attribute key is "`db.query.text`". This attribute should only be included if positional or named parameters are in use.
```

Comment:
Why does this omit the query based on there being named parameters?  In the context of spans, it seems like the query text should always be included.